Lib/statistics.py

```diff
@@ -952,11 +952,16 @@ def linear_regression(regressor, dependent_variable, /):
         raise StatisticsError('linear regression requires that both inputs have same number of data points')
     if n < 2:
         raise StatisticsError('linear regression requires at least two data points')
+    x, y = regressor, dependent_variable
+    xbar = fsum(x) / n
+    ybar = fsum(y) / n
+    sxy = fsum((xi - xbar) * (yi - ybar) for xi, yi in zip(x, y))
```

Comment:
Question: isn't the generator + zip going to make it slightly slower?

Reply:
That was an existing line take from *covariance()*.  I think it is the fastest way the run this computation.